### PR TITLE
make trace.TraceContext as default context propagator

### PR DESCRIPTION
**Pull request by clsung**
- for #312

### CODE REVIEW

File: api/trace/trace_context_propagator.go [hunk 1]
```diff
@@ -42,6 +42,9 @@ type TraceContext struct{}
 var _ propagation.TextFormat = TraceContext{}
 var traceCtxRegExp = regexp.MustCompile("^[0-9a-f]{2}-[a-f0-9]{32}-[a-f0-9]{16}-[a-f0-9]{2}-?")
 
+// DefaultPropagator is the default TraceContext propagator
```
Comment (krnowak) [inline api/trace/trace_context_propagator.go]: I'd like to reword it a bit:
```suggestion
// DefaultPropagator is the default trace propagator.
```

File: plugin/othttp/handler.go [hunk 2]
```diff
@@ -130,7 +130,7 @@ func NewHandler(handler http.Handler, operation string, opts ...Option) http.Han
 	h := Handler{handler: handler, operation: operation}
 	defaultOpts := []Option{
 		WithTracer(global.TraceProvider().Tracer("go.opentelemetry.io/plugin/othttp")),
-		WithPropagator(trace.TraceContext{}),
+		WithPropagator(trace.DefaultPropagator),
```
Comment (krnowak) [inline plugin/othttp/handler.go]: There is another mention of TraceContext in this file, in a comment. Could you update it to `DefaultPropagator` too?

Comment (clsung) [reply plugin/othttp/handler.go]: done

File: api/trace/trace_context_propagator.go [hunk 3]
```diff
@@ -42,6 +42,9 @@ type TraceContext struct{}
 var _ propagation.TextFormat = TraceContext{}
 var traceCtxRegExp = regexp.MustCompile("^[0-9a-f]{2}-[a-f0-9]{32}-[a-f0-9]{16}-[a-f0-9]{2}-?")
 
+// DefaultPropagator is the default trace propagator.
+var DefaultPropagator propagation.TextFormat = TraceContext{}
```
Comment (rghetia) [inline api/trace/trace_context_propagator.go]: It is better to have a method than a var.

Comment (clsung) [reply api/trace/trace_context_propagator.go]: modified.

**Pull request merged**